network-api/networkapi/wagtailpages/models.py

```diff
@@ -517,6 +520,63 @@ class NewsPage(PrimaryPage):
     template = 'wagtailpages/static/news_page.html'
 
 
+class BlogPageTag(TaggedItemBase):
+    content_object = ParentalKey('wagtailpages.BlogPage', on_delete=models.CASCADE, related_name='tagged_items')
+
+
+class BlogPage(FoundationMetadataPageMixin, Page):
+    template = 'wagtailpages/blog_page.html'
+
+    parent_page_types = [
+        'MiniSiteNameSpace',
+    ]
+
+    subpage_types = []
+
+    author = models.CharField(
+        verbose_name='Author',
+        max_length=50,
+        blank=False,
+    )
+
+    body = StreamField([
+        ('paragraph', blocks.RichTextBlock(
+            features=[
+                'bold', 'italic',
+                'h2', 'h3', 'h4', 'h5',
+                'ol', 'ul',
+                'link', 'hr',
+            ]
+        )),
+        ('image', customblocks.AnnotatedImageBlock()),
+        ('image_text', customblocks.ImageTextBlock()),
+        ('image_text_mini', customblocks.ImageTextMini()),
+        ('video', customblocks.VideoBlock()),
+        ('linkbutton', customblocks.LinkButtonBlock()),
+        ('spacer', customblocks.BootstrapSpacerBlock()),
+        ('quote', customblocks.QuoteBlock()),
+    ])
+
+    # Editor panels configuration
+
+    content_panels = Page.content_panels + [
+        FieldPanel('author'),
+        StreamFieldPanel('body'),
+    ]
+
+    # Promote panels configuration
+    tags = ClusterTaggableManager(through=BlogPageTag, blank=True)
+
+    promote_panels = Page.promote_panels + [
+        FieldPanel('tags'),
+    ]
+
+    # Database fields
+
+    zen_nav = True
+    narrowed_page_content = True
```

Comment:
Are these flags still relevant?

Reply:
I'm removing `narrowed_page_content`. Not sure about `zen_nav` as [primary_nav.html](https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/templates/partials/primary_nav.html) checks `page.zen_nav` in multiple places in the file. Not sure how I can balance "redundant database meta" vs "messy template code". Suggestions?